### PR TITLE
Refactor EAS api to be consistent

### PR DESCRIPTION
**Pull request by kmacgugan**
Use filter=status:blah for all calls
Removes health from sidebar call
disconnected is also a status now for the api
Does not support multi select with disconnected status and another status

Both service group and service side bar calls will now look like this for status filters:
filter=status:critical
filter=status:warning
filter=status:ok
filter=status:unknown
filter=status:disconnected

Signed-off-by: kmacgugan <kmacgugan@chef.io>

### :athletic_shoe: How to Build and Test the Change
After starting all services, rebuild the applications service
Populate some data with `applications_populate_database`
Use status filters on the service groups table and service groups sidebar and see that they are giving you the correct entities in response. 

### :white_check_mark: Checklist

- [ ] Tests added/updated?
- [ ] Docs added/updated?

### :camera: Screenshots, if applicable

### CODE REVIEW

File: components/applications-service/integration_test/service_groups_filters_test.go [hunk 1]
```diff
@@ -223,32 +223,34 @@ func TestDisconnectedStatusFilter(t *testing.T) {
...
-		require.NoError(t, err)
-
-		req := &applications.ServiceGroupsReq{
-			Filter: []string{"connectedStatus:disconnected", "status:critical"},
-		}
-		response, err := suite.ApplicationsServer.GetServiceGroups(ctx, req)
-		assert.Nil(t, err)
-		assert.Len(t, response.ServiceGroups, 1)
-	})
+	// TODO: Re-enable this test when we support multiple selected filters
```
Comment (vjeffrey) [inline components/applications-service/integration_test/service_groups_filters_test.go]: is there a card to follow up on this todo?

File: components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts [hunk 2]
```diff
@@ -68,15 +68,15 @@ export class ServiceGroupsRequests {
     const options = {
       params: this.buildServicesBySGFilterParams(filters)
     };
-
+    console.log("object: %O", options)
```
Comment (vjeffrey) [inline components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts]: ? accidentally included in the change?

File: components/applications-service/integration_test/service_groups_filters_test.go (hunk 1 above)
Comment (kmacgugan) [reply components/applications-service/integration_test/service_groups_filters_test.go]: No we don't have one yet, this idea is still in the ideation phase. @apriofrost has talked about making it a cross-automate design change on all tabs.

File: components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts (hunk 2 above)
Comment (kmacgugan) [reply components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts]: oops! yes it snuck in there.

**Pull request merged**